config_test.go

```diff
@@ -288,55 +212,18 @@ func TestCorsHead(t *testing.T) {
 			t.Errorf("do not contain %s", method)
 		}
 	}
-	if resp.Request.Method != "HEAD" {
-		t.Errorf("expected method HEAD, but got %v", resp.Request.Method)
+	if resp.Request.Method != "OPTIONS" {
```

Comment:
If you get a `HEAD` method?

Reply:
when is a cors request the client will do a OPTIONS request for preflight before the concrete request so  we we don't need to check HEAD or others request type